pages/services/index.vue

```diff
@@ -1,15 +1,12 @@
 <template>
-  <div>
-    <HeroBlock hero-classes="services">
+  <article>
+    <HeroBlock hero-classes="services waves blue-waves bg-blue-light !pb-20">
       <template #content>
-        <h1>Our Services</h1>
-        <p>
-          We turn ideas into industry-leading products. Our experts proactively
-          remedy issues with existing code and build innovations from the ground
-          up.
-        </p>
+        <h1 itemprop="name" class="leading-tight text-5xl lg:text-6xl">
+          Software consultants who build it right the first time, every time
+        </h1>
         <div>
-          <nuxt-link class="btn btn-red" to="/contact/">
+          <nuxt-link class="btn btn-red w-full lg:w-auto" to="/contact/">
             Work with us
           </nuxt-link>
```

Comment:
```suggestion
          <div class="flex justify-center mt-3 w-full lg:justify-start lg:mt-8">
            <nuxt-link class="btn btn-red w-full lg:w-auto" to="/contact/">
              Work with us
            </nuxt-link>
          </div>
```

I think we're missing some styles here. This is what we have around other buttons in HeroBlocks. Might be good to add.